test/blackbox-tests/test-cases/melange/empty-entries-target-dir-exists.t

```diff
@@ -1,20 +0,0 @@
-Test (modules) field can be left empty
```

Comment:
this test was just bad, the target dir was only getting created because the wrapper module was still being generated.